content/departments/security/tooling/trivy/5-3-0.md

```diff
@@ -0,0 +1,16 @@
+# Accepted CVEs for Sourcegraph 5.3.0
+
+| CVE ID                                                                                                                   | Affected Images                                                                                                                                                                                                     | CVE Severity | CVSS Base Score | [Sourcegraph Assessment](../../../engineering/dev/policies/vulnerability-management-policy.md#severity-levels) | CVSS Environmental Score                                                                                                                                                | Details                                                                                                                                                                                                                                                      |
+| ------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------ | --------------- | -------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [CVE-2023-39325](https://nvd.nist.gov/vuln/detail/CVE-2023-39325)                                                        |                                                                                                                                                                                                                     | High         | 7.5             | Medium                                                                                                         | 4.7                                                                                                                                                                     | The services that are vulnerable to this issue are typically not exposed on the internet. The likelihood of exploitation is low and this does not have a significant impact on the security of the instance. The issue is not present in Sourcegraph itself. |
```

Comment:
the affected images column is empty here?

Reply:
fixed!